app/controllers/middleware_server_controller.rb

```diff
@@ -12,84 +12,109 @@ class MiddlewareServerController < ApplicationController
+    :middleware_server_resume  => {
+      :op           => :resume_middleware_server,
+      :log_timeline => 'MwServer.Resume.UserRequest',
+      :skip         => true,
+      :hawk         => N_('resuming'),
+      :msg          => N_('Resume')
     },
-    :middleware_dr_generate    => {:op   => :generate_diagnostic_report,
-                                   :skip => false,
-                                   :hawk => N_('generating JDR report'),
-                                   :msg  => N_('Generate JDR report')}
+    :middleware_dr_generate    => {
+      :op   => :generate_diagnostic_report,
+      :skip => false,
+      :hawk => N_('generating JDR report'),
+      :msg  => N_('Generate JDR report')
+    }
   }.freeze
 
   STANDALONE_ONLY = {
-    :middleware_server_stop     => {:op   => :stop_middleware_server,
-                                    :skip => true,
-                                    :hawk => N_('stopping'),
-                                    :msg  => N_('Stop')
+    :middleware_server_stop     => {
+      :op           => :stop_middleware_server,
+      :log_timeline => 'MwServer.Shutdown.UserRequest',
+      :skip         => true,
+      :hawk         => N_('stopping'),
+      :msg          => N_('Stop')
     },
-    :middleware_server_shutdown => {:op    => :shutdown_middleware_server,
-                                    :skip  => true,
-                                    :hawk  => N_('shutting down'),
-                                    :msg   => N_('Shutdown'),
-                                    :param => :timeout
+    :middleware_server_shutdown => {
+      :op           => :shutdown_middleware_server,
+      :log_timeline => 'MwServer.Shutdown.UserRequest',
+      :skip         => true,
+      :hawk         => N_('shutting down'),
+      :msg          => N_('Shutdown'),
+      :param        => :timeout
     },
-    :middleware_server_restart  => {:op   => :restart_middleware_server,
-                                    :skip => true,
-                                    :hawk => N_('restarting'),
-                                    :msg  => N_('Restart')
+    :middleware_server_restart  => {
+      :op           => :restart_middleware_server,
+      :log_timeline => 'MwServer.Restart.UserRequest',
+      :skip         => true,
+      :hawk         => N_('restarting'),
+      :msg          => N_('Restart')
     },
-    :middleware_add_deployment  => {:op    => :add_middleware_deployment,
-                                    :skip  => false,
-                                    :hawk  => N_('Not deploying to Hawkular server'),
-                                    :msg   => N_('Deployment initiated for selected server(s)'),
-                                    :param => :file
+    :middleware_add_deployment  => {
+      :op    => :add_middleware_deployment,
+      :skip  => false,
+      :hawk  => N_('Not deploying to Hawkular server'),
+      :msg   => N_('Deployment initiated for selected server(s)'),
```

Comment:
Why are these two messages (`:hawk` and `:msg`) so different?

Reply:
I'm asking just out of curiosity, other messages have also different meaning, so there's definitely a reason behind it.

Reply:
I think @rubenvp8510 would know more about the `:hawk´  But i think is the message showed if you try to execute this operation on the provider.